packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -237,6 +237,8 @@ export interface ISummaryRuntimeOptions {
     // and the root node when generating a summary if set to true.
     // Defaults to TRUE (disabled) for now.
     disableIsolatedChannels?: boolean;
+
+    maxOpsSinceLastSummary?: number;
```

Comment:
```suggestion

    // Defaults to 3000 ops
    maxOpsSinceLastSummary?: number;
```
Or some sort of comment/description of what this means.